test/Microsoft.SqlTools.ServiceLayer.UnitTests/Connection/ConnectionDetailsTests.cs

```diff
@@ -226,27 +235,25 @@ public void ConnectTimeoutShouldReturnNullIfSetToNull()
         }
 
         [Test]
-        public void SettingEncryptToStringShouldStillReturnBoolean()
+        public void SettingEncryptToStringShouldStillReturnEncryptOption()
         {
             ConnectionDetails details = new ConnectionDetails();
-
-            string encrypt = "True";
-            bool? expectedValue = true;
-            details.Options["encrypt"] = encrypt;
-
-            Assert.AreEqual(details.Encrypt, expectedValue);
+            details.Options["Encrypt"] = true.ToString();
+            Assert.AreEqual(details.Encrypt, true);
```

Comment:
`Assert.That(details.Encrypt, Is.True, "Encrypt should be true);`

(and same for below check). 

1. You should always add messages to failures when there's more than one assert so that it's easy to tell where it actually failed
2. You should try to use Constraint (fluent) assertions whenever possible. They're a lot easier to read, have better error messages and provide much more robust ways of checking results.